internal/test/issues/issue-1039/type-config-nullabletype-disabled.yaml

```diff
@@ -0,0 +1,11 @@
+package: issue1039nullable
+generate:
+  models: true
+  embedded-spec: true
+output-options:
+  skip-prune: true
+output: nullable-disabled/types.gen.go
```

Comment:
I think this would be clearer if this and the other package were named differently - `nullabledisabled` is a bit unclear at a first glance.

Maybe `defaultbehaviour`?